### PR TITLE
Add feature label for security test

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Add feature label for integration test TestMtlsStrictK8sCA, following the user guide
https://github.com/istio/istio/tree/master/pkg/test/framework/features#feature-coverage-reporting

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: tests/integration/security/mtlsk8sca/strict_test.go [hunk 1]
```diff
@@ -33,6 +34,7 @@ import (
 // - Send HTTP/gRPC requests between apps.
 func TestMtlsStrictK8sCA(t *testing.T) {
 	framework.NewTest(t).
+		Features(features.Security_Mtls_K8sCa).
```
Comment (therealmitchconnors) [inline tests/integration/security/mtlsk8sca/strict_test.go:37]:
```suggestion
		Features("security.mtls.k8s-ca").
```

Comment (therealmitchconnors) [reply tests/integration/security/mtlsk8sca/strict_test.go:37]: This looks great!  I've adjusted your feature label to reflect that there are no constants generated in the features package.  Any remaining constants should be removed, and I'll try to get to that tomorrow.

File: pkg/test/framework/features/features.yaml [hunk 2]
```diff
@@ -29,6 +29,8 @@ features:
       status:
   # features that allow users to secure their services and service mesh.
   security:
+    mtls:
+      k8s-ca:
```
Comment (howardjohn) [inline pkg/test/framework/features/features.yaml:33]: Can we put a comment about what this actually means?

This feature is actually not related at all to mtls, its used for 1 way TLS to istiod (currently)?

Comment (JimmyCYJ) [reply pkg/test/framework/features/features.yaml]: I have moved it to control-plane.
